api/src/controllers/message.controller.ts

```diff
@@ -37,30 +37,53 @@ class MessageController {
       });
       return;
     }
-    
+
     if(validator.isEmpty(request.body.body)) {
       response.status(400).json({
         error: 'Message body is required.',
       });
       return;
     }
-    
-    try {
-      const { id } = session;
-      const { body } = request.body;
-      const placeId = Number.parseInt(request.params.placeId);
-      const [messageId] = await db.message
-        .insert({
-          body,
-          member_id: id,
-          place_id: placeId,
+    const bannedwords = /(nigger)|(chinc)/i;
```

Comment:
Need more words to be added to list (add by putting words in parenthesis and putting vertical bar between them.

Reply:
Haha, even reading the words currenly in the banlist caused me to jump a bit :)